space-cli/cmd/modules/ingress/getters.go

```diff
@@ -31,6 +32,68 @@ func GetIngressRoutes(project, commandName string, params map[string]string) ([]
 		if err != nil {
 			return nil, err
 		}
+
+		if len(filters) > 0 {
+			for _, filter := range filters {
+				arr := strings.Split(filter, "=")
+				if len(arr) != 2 {
+					continue
+				}
```

Comment:
There can be multple `=` in the filter. You will have to join back the remaining